src/App.js

```diff
@@ -50,7 +50,9 @@ function App() {
         <input type="submit" value="Submit" />
       </form>
       {/* Is a <div> here for textOutput without a <p> tag or some other element as accessible and semantic? */}
-      <div id="result">{textOutput}</div>
+      <div id="result">
+        <p aria-label="text output">{textOutput}</p>
```

Comment:
`<p>` are a lot more descriptive than a generic `<div>`. Glad you thought about accessibility here too with the aria label!